Library/Homebrew/cmd/update-report.rb

```diff
@@ -197,6 +197,26 @@ def output_update_report
       end
     end
 
+    if Homebrew::EnvConfig.install_from_api?
+      api_cache = Homebrew::API::HOMEBREW_CACHE_API
+      [:formula, :cask].each do |type|
+        names_txt = api_cache/"#{type}_names.txt"
+        names_before_txt = api_cache/"#{type}_names.before.txt"
```

Comment:
I assume this is from the last time update or autoupdate was run, right?

Reply:
Yeh, that's the plan.